locales/de.json

```diff
@@ -47,7 +47,7 @@
 	"passenger-request-price.set_price": "In dieser App setzt der Fahrgast den Preis fest 😉 Gebe uns deinen Preis an (in lokaler Währung). Wie viel bist du bereit zu zahlen? (Oder gebe \"0\" (Null) an, wenn du es nicht weisst)",
 	"passenger-request-price.should_be_numeric": "Hoppla, der Preis sollte der numerische Wert (und in lokaler Währung) sein! Zum Beispiel: \"50\". Bitte versuche es erneut!",
```

Comment:
grammar error: "der numerische Wert (und in lokaler Währung)" -> "ein numerischer Wert (in lokaler Währung)" .. take out "und" because the error .should_be_numeric isn't about currency, so this is only a reminder, but it sounds as if this is part of the cause for the message and might confuse.